doc/install.rst

```diff
@@ -14,8 +14,9 @@ Dependencies
 * ``pandas`` (>=0.24.0, optional, for generating event statistics)
 * ``matplotlib`` (>=3.1.0, optional, for using the interactive data inspector)
 
-We recommend the `Anaconda <https://www.anaconda.com/download/>`_ Python
-distribution. We require that you use Python 3.7 or higher.
+We recommend the `Anaconda <https://www.anaconda.com/download/>`_ Python distribution,
+and that you install ``mne_bids`` in a dedicated Python environment.
```

Comment:
why so? I think requiring people to install a software in a dedicated Python environment is not a great practice IMO. MNE-Python needs it because it has grown a lot and has 3D viz etc, but MNE-BIDS is a relatively light package

Reply:
fair point, I'll remove the point from the general installation instructions and add it only to "how to set up a dev environment" --> because for a mne-bids dev environment, having an isolated env can help debugging (no interactions with other packages or software)